containers/rust-postgres/README.md

```diff
@@ -0,0 +1,78 @@
+# Rust & PostgreSQL
+
+## Summary
+
+*Develop applications with Rust and PostgreSQL. Includes a Rust application container and PostgreSQL server.*
+
+| Metadata | Value |
+|----------|-------|
+| *Contributors* | The VS Code Team |
+| *Categories* | Core, Languages |
+| *Definition type* | Docker Compose |
+| *Published images* | mcr.microsoft.com/vscode/devcontainers/rust |
+| *Available image variants* | buster, bullseye ([full list](https://mcr.microsoft.com/v2/vscode/devcontainers/rust/tags/list)) |
+| *Published image architecture(s)* | x86-64, arm64/aarch64 for `bullseye` variant |
+| *Works in Codespaces* | Yes |
+| *Container host OS support* | Linux, macOS, Windows |
+| *Container OS* | Debian |
+| *Languages, platforms* | Rust |
+
+## Using this definition
+
+This definition creates two containers, one for Rust and one for PostgreSQL. VS Code will attach to the Rust dev container, and from within that container the PostgreSQL container will be available on **`localhost`** port 5432. The default database is named `postgres` with a user of `postgres` whose password is `postgres`, and if desired this may be changed in `docker-compose.yml`. Data is stored in a volume named `postgres-data`.
```

Comment:
```suggestion
This definition creates two containers, one for Rust and one for PostgreSQL. VS Code will attach to the Rust dev container, and from within that container the PostgreSQL container will be available on **`localhost`** port 5432. The `.env` file sets the default credentials. The default database is named `postgres` with a user of `postgres` whose password is `postgres`, and if desired this may be changed in `docker-compose.yml`, `.env` and `.devcontainer.json`. Data is stored in a volume named `postgres-data`.
```